.github/workflows/check_release_tag.py

```diff
@@ -1,16 +1,32 @@
 # -*- coding: utf-8 -*-
 """Check that the GitHub release tag matches the package version."""
 import argparse
-import json
+import ast
+from pathlib import Path
+
+
+def get_version_from_module(content: str) -> str:
+    """Get the __version__ value from a module."""
+    # adapted from setuptools/config.py
+    try:
+        module = ast.parse(content)
+    except SyntaxError as exc:
+        raise IOError(f'Unable to parse module: {exc}')
+    try:
+        return next(
+            ast.literal_eval(statement.value) for statement in module.body if isinstance(statement, ast.Assign)
+            for target in statement.targets if isinstance(target, ast.Name) and target.id == '__version__'
+        )
+    except StopIteration:
+        raise IOError('Unable to find __version__ in module')
+
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
     parser.add_argument('GITHUB_REF', help='The GITHUB_REF environmental variable')
-    parser.add_argument('SETUP_PATH', help='Path to the setup.json')
+    parser.add_argument('INIT_PATH', help='Path to the aiida/__init__.py file')
```

Comment:
Does this deserve an option? Are we ever going to change the location? Would simplify this to being hardcoded

Reply:
ok removed